conf/services.conf

```diff
@@ -81,8 +81,7 @@ simple_api_endpoints = {
             PhysicalDescription = ^ca_objects.PhysicalDescription,
             Subjects = "["<unit relativeTo="ca_occurrences" delimiter="," restrictToTypes="subject">&quot;^preferred_labels&quot;</unit>"]",
             Summary = ^ca_objects.Summary,
-            Notes = ^ca_objects.Notes,
-            TermsOfUse = ^ca_objects.TermsOfUse
```

Comment:
Heh, sorry it was late and if some that change several hours earlier, so when I commented I was misreading  the diff. I guess the right way to check it's correct is to check the requirements doc.